libcob/intrinsic.c

```diff
@@ -1295,7 +1295,7 @@ cob_intr_day_of_integer (cob_field *srcdays)
 	int		leapyear = 365;
 	cob_field_attr	attr;
 	cob_field	field;
-	char		buff[16];
+	char		buff[19];
```

Comment:
Even if the size of buff is 18, the warning messages will be solved.
Set the size of buff to 18.